source/Mission.cpp

```diff
@@ -639,6 +636,9 @@ bool Mission::CanComplete(const PlayerInfo &player) const
 // bright or dim text colors.
 bool Mission::IsSatisfied(const PlayerInfo &player) const
 {
+	if(!toComplete.Test(player.Conditions()))
```

Comment:
Move this check after the waypoint & stopover checks, as it is more expensive.

Reply:
The outfits and fines checks may be cheaper too.  NPCs may be expensive though.  Maybe it should go between outfits/fines and NPCs?